src/conventional/amv_ssec_ascii2ioda.py

```diff
@@ -0,0 +1,323 @@
+            try:
+                year = int(row['day'][0:4])
+                month = int(row['day'][5:6])
+                day = int(row['day'][7:8])
+                hour = int(row['hms'][0:1])
+                minute = int(row['hms'][2:3])
+                second = 0
+                dtg = datetime(year, month, day, hour, minute, second)
+                time_offset = np.int64(round((dtg - epoch).total_seconds()))
+                data['dateTime'] = np.append(data['dateTime'], time_offset)
+                data['longitude'] = np.append(data['longitude'], float(row['lon']))
+                data['latitude'] = np.append(data['latitude'], float(row['lat']))
+
+                freq, unk_freq = get_frequency(row['type'], unk_freq)
+                satid, unk_sat = get_id(row['sat'], unk_sat)
+                data['satelliteID'] = np.append(data['satelliteID'], satid)
+                data['sensorCentralFrequency'] = np.append(data['sensorCentralFrequency'], freq)
+                data['sensorZenithAngle'] = np.append(data['sensorZenithAngle'], float(row['rff']))
+                data['windTrackingCorrelation'] = np.append(data['windTrackingCorrelation'], float(row['qi']))
+                data['windHeightAssignMethod'] = np.append(data['windHeightAssignMethod'], int(row['int']))
+
+                pres = float(row['pre'])*100.
+                data['air_pressure'] = np.append(data['air_pressure'], pres)
+                wdir = float(row['dir'])*1.0
+                wspd = float(row['spd'])*1.0
+                if (wdir >= 0 and wdir <= 360 and wspd >= 0 and wspd < 300):
+                    uwnd, vwnd = meteo_utils.meteo_utils().dir_speed_2_uv(wdir, wspd)
+                else:
+                    uwnd = float_missing_value
+                    vwnd = float_missing_value
+
+                data['eastward_wind'] = np.append(data['eastward_wind'], uwnd)
+                data['northward_wind'] = np.append(data['northward_wind'], vwnd)
+
+            except KeyError as e:
+                if not keyerr:
+                    logging.warning(file_name + ' is missing variable ' + e.args[0])
+                keyerr = True
+
+                if (e.args[0] == 'dir') | (e.args[0] == 'spd'):
+                    data['eastward_wind'] = np.append(data['eastward_wind'], float_missing_value)
+                    data['northward_wind'] = np.append(data['northward_wind'], float_missing_value)
+                else:
+                    outname, missing = get_outname(e.args[0])
+                    data[outname] = np.append(data[outname], missing)
+
+        for f in unk_freq:
+            logging.warning(file_name + ' contains unknown frequency ' + f)
+
+        for s in unk_sat:
+            logging.warning(file_name + ' contains unknown satellite ID ' + s)
+
+    return data
+
+
+def get_frequency(obs_type, unk_freq):
+
+    if obs_type == 'IR':
+        freq = 2.99792458E+14/10.7
+    elif (obs_type == 'WVCA') | (obs_type == 'WVCT') | (obs_type == 'WV'):
```

Comment:
I believe my change here makes more sense to test simply if "WV" in obs_type rather than 3 specific choice letter combinations.

Reply:
If you decide to keep the three conditional tests (as code is now), please change the bitwise or operators ('|') to logical or operators ('or'). Ie,
```suggestion
    elif (obs_type == 'WVCA') or (obs_type == 'WVCT') or (obs_type == 'WV'):
```

Reply:
I have removed this check in favor of a known frequency dictionary